pkgs/by-name/ko/koboldcpp/package.nix

```diff
@@ -31,15 +32,23 @@
   vulkan-loader,
 
   metalSupport ? stdenv.isDarwin && stdenv.isAarch64,
+  march ? "",
+  mtune ? "",
 }:
 
 let
   makeBool = option: bool: (if bool then "${option}=1" else "");
 
-  makeWrapperArgs = lib.optionalString config.cudaSupport ''
+  libraryPathWrapperArgs = lib.optionalString config.cudaSupport ''
     --prefix LD_LIBRARY_PATH: "${lib.makeLibraryPath [ addDriverRunpath.driverLink ]}"
   '';
 
+  darwinFrameworks =
+    if (stdenv.isDarwin && stdenv.isx86_64) then
+      darwin.apple_sdk.frameworks
+    else
+      darwin.apple_sdk_11_0.frameworks;
```

Comment:
Yet again I'm getting flashbacks with this, I feel like `darwin.apple_sdk.frameworks` already contains a similar logic...